### ISSUE TITLE
Update delta table properties

### ISSUE THREAD

**Issue by scottsand-db** [delta-io/delta-docs#41]
https://docs.delta.io/latest/table-properties.html

e.g. it is missing `checkpointInterval` and, I'm assuming, others

**Comment by Seokyun-Ha** [comment]
I agree with it too. I think this `checkpointInterval` property is missing.
